conf/messages

```diff
@@ -626,12 +626,18 @@ fh.businessPartner.vatNumber.label=VAT registration number
 fh.businessPartner.vatNumber.hintText=This is 9 numbers, sometimes with 'GB' at the start, for example 123456789 or GB123456789. You can find it on {0}’s VAT registration certificate.
 fh.business_partners.title.v2= Does the partner have a National Insurance number? - Business partners
 fh.business.partners.nino.h1 = Does {0} have a National Insurance number?
+fh.business.partners.find.address.button = Find UK address
+fh.business.partners.partnership.registered.address.heading=What is {0}’s registered office address?
+
```

Comment:
Missing error message for `fh.partnerAddressLine.error.pattern` check line 652 for error message